### ISSUE TITLE
Accessibility: MAS 2.4.7: Broken keyboard focus is visible in "Send popup" in new message screen

### ISSUE THREAD

**Issue by ankurlal2015** [microsoft/fluentui#15322]
### Environment Information

- **Package version(s)**: 
- **Browser and OS versions**: 
Windows OS version: 2004 (Build 20206.1000)
Edge dev version:  86.0.622.10 (Official build) dev (64-bit)

### Describe the issue:

Pre-requisites:
Open URL: https://outlook-sdf.office.com/ and sign in with valid credentials

Repro Steps:Invoke the 'New message' button and then invoke "Send" popup using Alt+down arrow.
Navigate through the dialog using arrow keys and observe the keyboard focus.

Actual Result:
The keyboard focus appears as a single dotted line above the currently selected element instead of a boundary for the element. As there are only two elements the focus is not clearly known on which button the focus is present.
Note: The issue is also observed in outlook live (https://outlook.live.com/)

Expected Result:
The keyboard focus should be clearly visible as a complete boundary(rectangle) for the element.

User Impact:
Users with motor impairment who rely on keyboard will not be able to understand which element is currently focused if the focus is not visible or it appears as a single line instead of a complete boundary.

MAS Link: MAS 2.4.7 - Focus Visible

**Comment by paulgildea** [comment]
@ankurlal2015 Thanks for filing the issue.

Could provide an isolated reduced repro that we could take a look at? That would be super helpful because that can let us know if we've introduce a regression especially having the version of the library that's being used.

After running through the repro steps I'm assuming you are referring to this piece of the UI:
![image](https://user-images.githubusercontent.com/33229964/94943509-ada2eb00-048c-11eb-97b4-7c029b465a4e.png)

When I look at the isolated sample the [documentation site](https://developer.microsoft.com/en-us/fluentui#/controls/web/button), I see the following behavior pressing `Alt`+`Down Arrow` on a `SplitButton`:
![image](https://user-images.githubusercontent.com/33229964/94943706-f9ee2b00-048c-11eb-98fb-314872548e05.png)

This leads me to believe that there might be something in the app that is overriding the default focus visuals.
